openedx/core/djangoapps/programs/management/commands/backpopulate_program_credentials.py

```diff
@@ -95,7 +140,7 @@ def _flatten(self, programs):
 
         return course_runs
 
-    def _load_usernames(self):
+    def _load_usernames(self, users=None):
```

Comment:
Same here `usernames`.

Reply:
Changing the parameter field `users` to `usernames` can make the code confusing as there is already a field `self.usernames` being used in the function body.